pkg/pxc/app/statefulset/node.go

```diff
@@ -182,25 +220,29 @@ func (c *Node) PMMContainer(spec *api.PMMSpec, secrets string, v120OrGreater boo
 			Value: "--query-source=perfschema",
 		},
 	}
-
-	clusterEnvs := []corev1.EnvVar{
-		{
-			Name:  "DB_CLUSTER",
-			Value: app.Name,
-		},
-		{
-			Name:  "DB_HOST",
-			Value: "localhost",
-		},
-		{
-			Name:  "DB_PORT",
-			Value: "3306",
-		},
-	}
-
 	ct.Env = append(ct.Env, pmmEnvs...)
-	if v120OrGreater {
+
+	if versionGreaterOrEqual120 {
```

Comment:
no need in an intermediate variable here, just `compare >= 1`. but `compare` should be renamed